channel/src/main/java/com/synopsys/integration/alert/channel/jira/common/distribution/delegate/JiraIssueCreator.java

```diff
@@ -69,6 +70,8 @@ protected JiraIssueCreator(
             return new ExistingIssueDetails<>(createdIssue.getId(), createdIssue.getKey(), createdIssueFields.getSummary(), issueUILink);
         } catch (IntegrationRestException restException) {
             throw jiraErrorMessageUtility.improveRestException(restException, issueCreatorDescriptorKey, extractReporter(creationRequest));
+        } catch (JiraPreconditionNotMetException jiraException) {
+            throw new AlertException(String.format("Failed to create an issue in Jira. %s", jiraException.getMessage()), jiraException);
```

Comment:
Repeating the string "Failed to create an issue in Jira." might be considered a code-smell by SonarCloud. Can we pull it out into a private constant just to be safe?